packages/ui/react-ui/src/CrossmintPayButton.tsx

```diff
@@ -42,6 +42,9 @@ export const CrossmintPayButton: FC<CrossmintPayButtonReactProps> = ({
     prepay,
     locale = "en-US",
     currency = "USD",
+    layout = "popUpWindow",
+    successCallbackURL = "https://www.crossmint.com/console/",
```

Comment:
do we need fallbacks? we obviously dont want to redirect to the console after a user has checkout out

the flow should remain the same as it was before these changes, if no callbackURLs are passed